<?xml version="1.0" encoding="utf-8"?>

<launch>

    <!-- # args -->
    <arg name="topic_gps" default="/gps_fix" />
    <arg name="path_file" default="$(find ros_gps_nav)/files/nav_setup.csv" />
    <arg name="param_nav_setup" default="/nav_setup" />
    <arg name="is_pub_status" default="true" />

    <!-- # gps recorder -->
    <node pkg="ros_gps_nav" type="gps_rec.py" name="gps_rec" output="screen">
        <param name="topic_gps" value="$(arg topic_gps)" />
        <param name="path_file" value="$(arg path_file)" />
        <param name="param_nav_setup" value="$(arg param_nav_setup)" />
        <param name="is_pub_status" value="$(arg is_pub_status)" />
    </node>

</launch>
